src/Test/Inject/TestInjector.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from "react";
+import { AuthContext } from "@/Data/Auth/AuthContext";
+import { Dependencies, DependencyProvider } from "@/UI";
+
+type Props = {
+  // Define your component props here
+  dependencies: Partial<Dependencies>;
+};
+
+export const TestInjector: React.FC<React.PropsWithChildren<Props>> = ({
```

Comment:
comments please, what's the purpose of this injector? I see you're only using it in a few tests concerning authentication.